content/docs/fundamentals/converting/resolving-errors@pt_BR.md

```diff
@@ -0,0 +1,350 @@
+
+{% call callout('Observação', type='note') %}
+O estilo in-line é obrigatório. Além disso, há um limite de tamanho de arquivo de 50 kilobytes para todas as informações de estilo. Use pré-processadores de CSS, como o [SASS](http://sass-lang.com/) (em inglês), para reduzir o CSS antes de realizar a inserção in-line dele nas páginas AMP.
+{% endcall %}
+
+{% call callout('Importante', type='caution') %}
+É possível ter somente uma tag de estilo em todo o documento AMP. Se várias folhas de estilo externas forem referenciadas pelas páginas AMP, será necessário agrupá-las em um único conjunto de regras. Para saber quais regras de CSS são válidas na AMP, leia [CSS compatível](/pt_br/docs/design/responsive/style_pages.html).
+{% endcall %}
+
+## Excluir JavaScript de terceiros
+
+É relativamente fácil alterar as folhas de estilo inserindo CSS in-line, mas isso não se aplica ao JavaScript.
+
+<pre class="error-text">
+A tag "script" não é permitida, exceto em formulários específicos.
+</pre>
+
+Em geral, os scripts só são permitidos na AMP quando cumprem dois requisitos principais:
+
+1.  Todo JavaScript precisa ser assíncrono (ou seja, incluir o atributo `async` na tag de script).
+2.  O JavaScript é para a biblioteca AMP e para qualquer componente AMP na página.
+
+Isso exclui efetivamente o uso de todo JavaScript gerado por usuários/terceiros na AMP, exceto no caso indicado abaixo.
+
+{% call callout('Observação', type='note') %}
+As únicas exceções à restrição a scripts gerados por usuários/terceiros são:
+
+1.  Scripts que adicionam metadados à página ou configuram componentes AMP: terão o atributo de tipo `application/ld+json` ou `application/json`.
+2.  Scripts incluídos em iframes:  só inclua JavaScript em um iframe em último caso. Sempre que possível, substitua os recursos JavaScript usando os [Componentes AMP](/pt_br/docs/reference/components.html). Veremos nosso primeiro componente AMP na próxima seção.
+{% endcall %}
+
+Tente abrir o arquivo externo [`base.js`] (https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.js). O que você vê? O arquivo deverá estar vazio, sem qualquer código JavaScript, e incluir apenas um comentário com informações como esta:
+
+```javascript
+/*
+
+This external JavaScript file is intentionally empty.
+
+Its purpose is merely to demonstrate the AMP validation error related to the
+use of external JavaScript files.
+
+*/
+```
+
+Considerando que esse arquivo JavaScript externo não é um componente funcional do site, é possível remover a referência inteira com segurança.
+
+**Remova** a seguinte referência externa de JavaScript do documento:
+
+```html
+<script type="text/javascript" src="base.js"></script>
+```
+
+Agora, **atualize** a página e verifique se o erro do script desapareceu.
+
+## Incluir CSS padrão da AMP
+
+Os seguintes erros se referem à ausência de código boilerplate:
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).